packages/protocol/src/defaults/common-defaults.ts

```diff
@@ -186,6 +187,11 @@ export const DEFAULT_WSREQUEST_DATA: WsRequestData = {
   properties: {}
 } as const;
 
+export const DEFAULT_PROCESS_DATA: ProcessConfig = {
+  data: '',
+  permissions: { view: { allowed: true } }
+};
```

Comment:
I think this is not necessary, as the DEFAULT_WSPROCESS_DATA already defines this default values